src/constants/InitialVentilatorConfiguration.ts

```diff
@@ -1,12 +1,12 @@
-const initalVentilatorConfiguration = {
-  'Ventilation Mode': 'PCV',
-  'I:E Ratio': '1:1',
-  'Respiratory Rate': 12,
-  'Tidal Volume': 200,
-  'Pressure': 15,
-  'Flow Trigger': 1,
-  'PEEP': 0,
-  'FiO2': '20 - 30',
-}
+const initialVentilatorConfiguration = {
+  ventilationMode: 'PCV',
+  ieRatio: '1:1',
+  respiratoryRate: 12,
+  tidalVolume: 200,
+  pressure: 15,
+  flowTrigger: 1,
+  peep: 0,
+  fiO2: '20 - 30',
```

Comment:
comma for the last field of the object, Is this change related to clean as that is the scope of this PR?

Reply:
yes, that is part of the prettier linting

Reply:
@themakshter can we add linting and formatting standards in docs so that contributors know that beforehand, sort of an on-boarding guide?

Reply:
Good point! Could you create an issue for it @aqilusman45 ?

Reply:
Sure